azure.yaml

```diff
@@ -1,3 +1,17 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
 
 name: kernel-memory
+workflows:
+  up:
+    steps:
+      - azd: provision
+hooks:
+  postprovision:
+    windows:
+      shell: pwsh
+      interactive: true
+      run: "Write-Host \"Kernel Memory is ready at: $env:kmServiceEndpoint"
```

Comment:
before merging, could you confirm that this will not cause errors? I see an escaped `"` after `Write-Host` but there is no corresponding quote at the end

Reply:
e.g. should this be
```
run: "Write-Host \"Kernel Memory is ready at: $env:kmServiceEndpoint\""
```
or
```
run: "Write-Host Kernel Memory is ready at: $env:kmServiceEndpoint"
```

Reply:
@vhvb1989